gradle.properties

```diff
@@ -19,14 +20,14 @@ org.gradle.vfs.watch=true
 org.gradle.jvmargs=-Xmx6g -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
 
 # kotlin
-kotlin_version=1.6.0
+kotlin_version=1.6.10
 kotlin.native.ignoreDisabledTargets=true
 kotlin.mpp.stability.nowarn=true
 kotlin.mpp.enableCompatibilityMetadataVariant=true
 
 # kotlin libraries
 serialization_version=1.3.0
-coroutines_version=1.5.2-native-mt
+coroutines_version=1.6.0-RC3
```

Comment:
1.6.0 is out :)

Reply:
Yep, will fix. The merge is blocked by the bug in GC :(

Reply:
In GC?
Or do you mean this bug: https://youtrack.jetbrains.com/issue/KT-50477? But this should not be hard to workaround :)